docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,27 @@
+<h2>How to Contribute:</h2>
+
+CAS CIIP portal is accepting contributions via Github pull requests. (If you're new to GitHub, you might start with a [basic tutorial](https://help.github.com/en/github/getting-started-with-github/set-up-git) and check out a more detailed guide to [pull requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests))This document outlines the contribution work flow, contact points for developers and other resources to make getting your contribution accepted.
+
+<h2>Certificate of Origin</h2>
+
+By contributing to this project you agree to the [Certificate of Origin (DCO)](https://github.com/deis/deis/blob/master/DCODeveloper). This document was created by the Linux Kernel community and is a simple statement that you, as a contributor, have the legal right to make the contribution.
+
+<h2>Support Channels</h2>
+
+Before opening a new issue, it's helpful to search the project - it's likely that another user has already reported the issue you're facing, or it's a known issue that we're already aware of.
+[Create An Issue](https://github.com/bcgov/BC-Policy-Framework-For-GitHub/issues/new?title=Devhub%20Issue%3A%20Sample%20Contributing%20File%20%5Bshort%20description%20here%5D&body=%3E%20path%3A%20(do%20not%20delete)%20https%3A%2F%2Fgithub.com%2Fbcgov%2FBC-Policy-Framework-For-GitHub%2Fblob%2Fmaster%2FBC-Gov-Org-HowTo%2FSAMPLE-CONTRIBUTING.md%0A%20%3E%20(do%20not%20delete)%20devhub%20page%3A%20%2FSample-Contributing-File%0A%0A%23%23%20Devhub%20Content%20Issue%0A)
```

Comment:
this looks like it's creating an issue on https://github.com/bcgov/BC-Policy-Framework-For-GitHub/issues ...did you mean to do that?